src/ReceivingList/ReceivingListFilter/ReceivingListFilter.js

```diff
@@ -118,6 +119,15 @@ const ReceivingListFilter = ({
         onChange={adaptedApplyFilters}
       />
 
+      <BooleanFilter
+        id={`filter-${FILTERS.RUSH}`}
+        activeFilters={activeFilters[FILTERS.RUSH]}
+        disabled={disabled}
+        labelId="ui-orders.filter.rush"
```

Comment:
Maybe, it's better to keep this translation in `ui-receiving`

Reply:
Maybe) Add translation.